src/main/java/com/onarandombox/MultiversePortals/listeners/MVPPlayerListener.java

```diff
@@ -237,6 +237,21 @@ public void playerPortal(PlayerPortalEvent event) {
                     event.setCancelled(true);
                     return;
                 }
+
+                Location destLocation = portalDest.getLocation(event.getPlayer());
+                if (destLocation == null) {
+                    Logging.fine("Portal event cancelled because destination is null!");
+                    event.setCancelled(true);
+                    return;
+                }
+
+                MultiverseWorld world = this.plugin.getCore().getMVWorldManager().getMVWorld(destLocation.getWorld().getName());
```

Comment:
Looks like `world` isn't use anywhere else. If this is the case you can just do something like
`if (!this.plugin.getCore().getMVWorldManager().isMVWorld(destLocation.getWorld())`

Reply:
Good point.